src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../mixins/button";
+@use "../../design-tokens/sizes.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ *
+ * 1. The block editor allows editors to adjust button border radius. This style
+ *    is applied via an inline style on the `wp-block-button__link` element. In
+ *    order to prevent unwanted border-radius we apply an `!important` tag.
```

Comment:
Could you move this comment so that it is above the block of CSS where the number exists rather than above its parent?